designer/client/src/components/tips/ValidTips.tsx

```diff
@@ -1,15 +1,19 @@
 import React from "react";
 import TestingMode from "../../assets/img/icons/testingMode.svg";
 import CheckCircleIcon from "@mui/icons-material/CheckCircle";
+import HourglassEmptyIcon from "@mui/icons-material/HourglassEmpty";
 import ValidTip from "./ValidTip";
 import { useTheme } from "@mui/material";
 
-export default function ValidTips(props: { hasNeitherErrorsNorWarnings?: boolean; testing?: boolean }): JSX.Element {
-    const { hasNeitherErrorsNorWarnings, testing } = props;
+export default function ValidTips(props: { loading: boolean; hasNeitherErrorsNorWarnings?: boolean; testing?: boolean }): JSX.Element {
+    const { loading, hasNeitherErrorsNorWarnings, testing } = props;
     const theme = useTheme();
 
     return (
         <React.Fragment>
+            {loading && (
+                <ValidTip icon={HourglassEmptyIcon} message={"Verifying scenario and loading tips"} color={theme.palette.warning.main} />
```

Comment:
To be consistent with the rest of the application, maybe we can use [CircularProgress](https://mui.com/material-ui/react-progress/#circular-indeterminate) instead?

Reply:
done
[Screencast from 02-07-2025 04:53:45 PM.webm](https://github.com/user-attachments/assets/3e846cae-7676-40b1-bf82-3f3453096eee)